ts/src/binance.ts

```diff
@@ -9091,4 +9093,211 @@ export default class binance extends Exchange {
             'info': interest,
         };
     }
+
+    async fetchMyLiquidations (symbol: string, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name binance#fetchMyLiquidations
+         * @description retrieves the users liquidated positions
+         * @see https://binance-docs.github.io/apidocs/spot/en/#get-force-liquidation-record-user_data
+         * @see https://binance-docs.github.io/apidocs/futures/en/#user-39-s-force-orders-user_data
+         * @see https://binance-docs.github.io/apidocs/delivery/en/#user-39-s-force-orders-user_data
+         * @param {string} symbol unified CCXT market symbol
+         * @param {int} [since] the earliest time in ms to fetch liquidations for
+         * @param {int} [limit] the maximum number of liquidation structures to retrieve
+         * @param {object} [params] exchange specific parameters for the binance api endpoint
+         * @param {int} [params.until] timestamp in ms of the latest liquidation
+         * @returns {object} an array of [liquidation structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#liquidation-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        let request = {
+            'symbol': market['id'],
+            'autoCloseType': 'LIQUIDATION',
+        };
+        if (since !== undefined) {
+            request['startTime'] = since;
+        }
+        if (limit !== undefined) {
+            if (market['spot']) {
+                request['size'] = limit;
+            } else {
+                request['limit'] = limit;
+            }
+        }
+        [ request, params ] = this.handleUntilOption ('endTime', request, params);
+        let response = undefined;
+        if (market['spot']) {
+            response = await this.sapiGetMarginForceLiquidationRec (this.extend (request, params));
```

Comment:
spot seems to accept a different symbol key 
![image](https://github.com/ccxt/ccxt/assets/43336371/57d8fcc0-2aa5-4192-9558-725aa9614ec2)